src/Confluent.Kafka/Confluent.Kafka.csproj

```diff
@@ -5,7 +5,10 @@
     <Description>Confluent's .NET Client for Apache Kafka</Description>
     <Copyright>Copyright 2016-2020 Confluent Inc., Andreas Heider</Copyright>
     <PackageProjectUrl>https://github.com/confluentinc/confluent-kafka-dotnet/</PackageProjectUrl>
-    <PackageLicenseUrl>https://github.com/confluentinc/confluent-kafka-dotnet/blob/master/LICENSE</PackageLicenseUrl>
+    <PackageLicenceExpression>Apache-2.0</PackageLicenceExpression>
```

Comment:
must be PackageLicenseExpression, see https://github.com/NuGet/Home/wiki/Packaging-License-within-the-nupkg#project-properties